CHANGELOG.md

```diff
@@ -20,6 +22,15 @@
 
 ### Miscellaneous
 
+* A pytest runner is shipped with every installation ([#1633](https://github.com/biocore/scikit-bio/pull/1633))
+
+* The nosetest framework has has been replace in favor of pytest ([#1624](https://github.com/biocore/scikit-bio/pull/1624))
+
+* The numpy docs are deprecated in favor of napoleon ([#1629](https://github.com/biocore/scikit-bio/pull/1629))
+
+* This version is now compatiable with pandas 0.23 ([#1627](https://github.com/biocore/scikit-bio/pull/1627))
```

Comment:
Can you also mention NumPy? Maybe mention the ranges themselves, something like:

` This version is now compatible with NumPy >= 1.9.2 and Pandas >= 0.23`.